flask/cli.py

```diff
@@ -590,7 +592,10 @@ def run_command(info, host, port, reload, debugger, eager_loading,
     """
     from werkzeug.serving import run_simple
 
-    debug = get_debug_flag()
+    debug = get_debug_flag(
+        info.load_app().config.get('DEBUG')
```

Comment:
No. This will break the reloader when loading the app fails e.g. due to a syntax error.